.github/workflows/dispatch-build-bottle.yml

```diff
@@ -289,16 +289,14 @@ jobs:
         env:
           GH_TOKEN: ${{secrets.HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN}}
         run: |
-          # Silence lint error about backtick usage inside single quotes.
-          # shellcheck disable=SC2016
           gh pr create \
```

Comment:
This is probably also a good candidate for extraction to a composite action.

Reply:
Agreed!